.github/workflows/bulk-dependency-updates.yaml

```diff
@@ -80,6 +80,13 @@ jobs:
           git commit -m "Automated dependency upgrades"
           git push -f origin ${{ github.ref_name }}:"$BRANCH_NAME"
 
+      - name: Get go-mod-changelog script
+        id: changelog
+        run: |
+          curl -o go-mod-changelog.py https://raw.githubusercontent.com/hashicorp/vault-workflows-common/main/go-mod-changelog.py
```

Comment:
Is there anything in the `github` event context we can use to replace hardcoding `main`, so that if someone calls the workflow with a non-main branch they get a consistent state?

Reply:
Good question! Not sure but I can look into it.

Reply:
It would probably be easiest to just checkout `vault-workflow-commons` repo?

Reply:
> checkout vault-workflow-commons repo?

I think in that case we would still need to know the branch name used as the ref for the reusable workflow. I think I am going to stick with `main` for now unless someone has a better idea.

I did find something in the github context that seemed like it might work but it appears there are open issues with it being set.